lib/scan.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+log.info ()
+{ # $@=message
+  printf "$(date +'%H:%M:%S') [\033[34m%s\033[0m] %s\n" "INFO" "${*}";
+}
+
+log.error ()
+{ # $@=message
+  printf "$(date +'%H:%M:%S') [\033[31m%s\033[0m] %s\n" "ERROR" "${*}";
+}
+
+git.ls_remote ()
+{
+  git ls-remote --symref origin HEAD \
+    | awk '/^ref:/{sub(/refs\/heads\//, "", $2); print $2}'
+}
```

Comment:
@SUSTAPLE117 ^ so this should simplify customer onboarding because it auto-detect the default branch (`main` / `master` / `develop`, ...)

Reply:
Nice!